src/main/java/bio/terra/cli/command/workspace/List.java

```diff
@@ -0,0 +1,48 @@
+package bio.terra.cli.command.workspace;
+
+import bio.terra.cli.auth.AuthenticationManager;
+import bio.terra.cli.context.GlobalContext;
+import bio.terra.cli.context.WorkspaceContext;
+import bio.terra.cli.service.WorkspaceManager;
+import bio.terra.workspace.model.WorkspaceDescription;
+import java.util.concurrent.Callable;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+/** This class corresponds to the third-level "terra workspace list" command. */
+@Command(
+    name = "list",
+    description = "List all workspaces the current user can access.",
+    showDefaultValues = true)
+public class List implements Callable<Integer> {
+
+  @CommandLine.Option(
+      names = "--offset",
+      required = false,
+      defaultValue = "0",
+      description =
+          "The offset to use when listing workspaces. (Zero means to start from the beginning.)")
+  private int offset;
```

Comment:
Why not add an option for the limit as well?

Reply:
You know, I initially thought that was maybe too much flexibility because why would someone want 100s of workspaces returned. But on second thought, seems like I can just make a reasonable default and then let users optionally override. I'll update this.